meinberlin/templates/base.html

```diff
@@ -49,7 +49,10 @@
     {% block header %}
       {% include 'header.html' %}
     {% endblock %}
-    <main id="main">
+    <!-- FIXME we don't currently want their grid in dashboard however this does not work if someone names wagtail, project or plan page 'dashboard' -->
```

Comment:
this is just temporary, right? otherwise we could have a templatetag which makes the check work better